backend/sirius-web-compatibility/src/main/java/org/eclipse/sirius/web/compat/services/diagrams/DiagramDescriptionConverter.java

```diff
@@ -71,4 +74,17 @@ public DiagramDescription convert(org.eclipse.sirius.diagram.description.Diagram
 
         return builder.build();
     }
+
+    private boolean isAutoLayoutMode(org.eclipse.sirius.diagram.description.DiagramDescription diagramDescription) {
+        boolean isAutoLayout = false;
+        EObject current = diagramDescription;
+        while (!isAutoLayout && current != null) {
+            if (current instanceof DocumentedElement) {
+                String doc = ((DocumentedElement) current).getDocumentation();
+                isAutoLayout = doc != null && doc.contains("FORCE_AUTO_LAYOUT"); //$NON-NLS-1$
```

Comment:
This could be extracted into a constant. Not sure it will be needed somewhere else, but the choice of name for that keyword become part of the API, so I'd rather have it as an explicit constant instead of a magic value inside a method.

Reply:
Done